### PR TITLE
chore: deps: update builtin-actors, GST, verified claims tests

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
As in the title:
- Update builtin-actors to v13.0.0
- Update GST to v0.13.1
- Update verified claims test

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

**Comment by aarshkshah1992** [comment]
@rvagg Removed the additional claims test you added as we're not shipping @LexLuthr 's extend claims CLI changes with this release.

**Review by Stebalien — APPROVED** [review]
Good to me, but I haven't checked the CIDs.

File: build/builtin_actors_gen.go [hunk 1]
```diff
@@ -249,8 +249,8 @@ var EmbeddedBuiltinActorsMetadata []*BuiltinActorsMetadata = []*BuiltinActorsMet
 }, {
 	Network:      "calibrationnet",
 	Version:      13,
-	BundleGitTag: "v13.0.0-rc.3",
-	ManifestCid:  MustParseCid("bafy2bzacea4firkyvt2zzdwqjrws5pyeluaesh6uaid246tommayr4337xpmi"),
+	BundleGitTag: "v13.0.0",
+	ManifestCid:  MustParseCid("bafy2bzacect4ktyujrwp6mjlsitnpvuw2pbuppz6w52sfljyo4agjevzm75qs"),
```
Comment (arajasek) [inline build/builtin_actors_gen.go:253]: This doesn't seem correct, we shouldn't be updating the binary for calibnet in this PR (or else we won't sync that network anymore). We should update for everything _except_ calibnet by running `./pack.sh v13 v13.0.0 calibrationnet=v13.0.0-rc.3`.

**Review by arajasek — CHANGES_REQUESTED** [review]

File: build/builtin_actors_gen.go (hunk 1 above)
Comment (rjan90) [reply build/builtin_actors_gen.go]: Pushed a fix in: https://github.com/filecoin-project/lotus/pull/11768/commits/f6e65bf0e5d2001c5d885f9b6b390eb3c5502861

**Review by arajasek — APPROVED** [review]
LGTM!

**Pull request merged**